### PR TITLE
Updated command to get environment related to issue #7156

### PR DESCRIPTION
**Pull request by vijay-wagento**
## Purpose of this pull request

This pull request (PR) created to update the command to get environments using magento-cloud command

## Affected DevDocs pages

https://devdocs.magento.com/cloud/reference/cli-ref-topic.html
-  ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/reference/cli-ref-topic.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The following list of commonly used Magento Cloud CLI commands includes required
 Command | Description
 ------- | -----------
 `magento-cloud login` | Log in to the project.
-`magento-cloud list` | List the environments in the current project.
```
Comment (rogyar) [inline src/cloud/reference/cli-ref-topic.md:62]: Hi @vijay-wagento. Thank you for the fix, it makes sense. 
Could I ask you to leave both commands but with the different descriptions, please? 

```
`magento-cloud list` | List the available commands for CLI tool.
`magento-cloud environment:list` | List the environments in the current project.
```

Thank you!

Comment (vijay-wagento) [reply src/cloud/reference/cli-ref-topic.md:62]: Hello, @rogyar I have added that command in the list as per suggest.

Comment (vijay-wagento) [reply src/cloud/reference/cli-ref-topic.md:62]: Done

**Comment by meker12** [comment]
@vijay-wagento Can you update this branch with master so that we can merge the PR? Thanks for the contribution!

**Comment by vijay-wagento** [comment]
I have merged master branch to my branch.

**Pull request merged**